metapool/metapool.py

```diff
@@ -2540,9 +2544,9 @@ def validate_plate_df(plate_df, metadata, sample_accession_df, blanks_dir,
     if missing_samples.empty:
         warnings.warn("All samples have associated metadata :D")
     else:
-        missing_str = ", ".join(missing_samples.astype(str))
+        missing_str = "', '".join(missing_samples.astype(str))
```

Comment:
I'm surprised that this didn't change any tests; maybe another way to ask this is: are there tests for this warning?

Reply:
Hah, yeah ... there is in fact a test (or rather an omnibus test with about 5 test cases packed into it) but what it tests is just that it received the expected error type (`ValueError`), not the contents of the error message:

https://github.com/biocore/metagenomics_pooling_notebook/blob/7052660b6f494fe805ca70df2e9f0447646a4c63/metapool/tests/test_metapool.py#L487-L492

Since this seems to bug you (too), I will take the opportunity to refactor.

Reply:
Well, *that* hurt more than it should have ... in case we needed a reminder that bare exception tests (without an expected message) is a bad idea, the fact that some of the tests turned out to be throwing `ValueError`s but not the ones they said they were testing for is a good reminder. Let's not do this again -_-